src/components/PlayerSearch.vue

```diff
@@ -34,17 +34,22 @@
           <p block style="margin:auto">Next</p>
         </v-btn>
       </div>
-      <div block v-for="(player, index) in players.result" :key="index" class="row">
-        <v-btn block
-          left
-          style="display:inline;
-          width:100%;
-          height:20;"
-          class="btn-text"
-          @click.native="$router.push(`/player/${player.uuid}`)">
-          <p block style="margin:auto">{{getRanks(player.uuid)}} {{player.lastSeenName}}</p>
-        </v-btn>
-      </div>
+      <v-list>
+        <v-list-item-group>
+          <v-list-item
+           v-for="(player, index) in players.result"
+           :key="index"
+           class="row"
+           @click.native="$router.push(`/player/${player.uuid}`)">
```

Comment:
I think you can use the `to` attribute instead to link to routes. That seems a bit cleaner.